src/clr-angular/emphasis/alert/alerts.ts

```diff
@@ -29,7 +29,7 @@ export class ClrAlerts implements AfterContentInit {
      */
     @Input("clrCurrentAlertIndex")
     public set _inputCurrentIndex(index: number) {
-        if (index) {
+        if (index || index === 0) {
```

Comment:
Can we also just simplify this to `index >= 0` instead of having an `||`?

Reply:
I didn't do that because `null >= 0` is `true`. With latest change I am checking Number.isInteger(index) && index >= 0
This should cover all the invalid values.